app/code/Magento/CatalogGraphQl/Model/Resolver/Product/StockCount.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\CatalogGraphQl\Model\Resolver\Product;
+
+use Magento\Catalog\Model\Product;
+use Magento\CatalogInventory\Model\Stock\Item;
+use Magento\Framework\GraphQl\Config\Element\Field;
+use Magento\Framework\GraphQl\Query\Resolver\Value;
+use Magento\Framework\GraphQl\Query\Resolver\ValueFactory;
+use Magento\Framework\GraphQl\Query\ResolverInterface;
+use Magento\Framework\GraphQl\Schema\Type\ResolveInfo;
+
+/**
+ * Resolve data for product stock count
+ */
+class StockCount implements ResolverInterface
+{
+    /**
+     * @var ValueFactory
+     */
+    private $valueFactory;
+
+    /**
+     * @var Item
+     */
+    private $item;
+
+    /**
+     * @param ValueFactory $valueFactory
+     * @param Item $item
+     */
+    public function __construct(
+        ValueFactory $valueFactory,
+        Item $item
```

Comment:
Stock item is not injectable and must not be injected to the constructor directly. Please inject ItemFactory instead and use it to instantiate an item inside of `resolve`